src/sentry/api/endpoints/project_transaction_anomaly_detection.py

```diff
@@ -0,0 +1,128 @@
+from datetime import datetime
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+from snuba_sdk import (
+    Column,
+    Condition,
+    Direction,
+    Entity,
+    Function,
+    Granularity,
+    Limit,
+    Offset,
+    Op,
+    OrderBy,
+    Query,
+)
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.utils.snuba import _snuba_pool
+
+
+class ProjectTransactionAnomalyDetectionEndpoint(ProjectEndpoint):
+    def get(self, request: Request) -> Response:
+        transaction = request.GET.get("transaction")
+        project_id = request.GET.get("project")
+        dataset = "transactions"
+        start, end = get_date_range_from_params(request.GET)
+        query_start, query_end, granularity = self.map_snuba_queries(
+            start.timestamp(), end.timestamp()
+        )
+
+        query = self.get_snuba_query(
+            dataset, query_start, query_end, granularity, project_id, transaction
+        )
+        snuba_transaction_data = self.submit_snuba_query(query)
+        return self.get_anomalies(snuba_transaction_data)
+
+    @staticmethod
+    def map_snuba_queries(start, end):
```

Comment:
this function takes start/end as integer timestamps, but we right now we are passing in datetimes (https://github.com/getsentry/sentry/blob/9a1e4a665c47d84fe607dad753229398a9524e1f/src/sentry/api/utils.py#L28)